### PR TITLE
docs: tranlsation of Checkbox

### PR DESCRIPTION
**Pull request by benjycui**
ref: #1471

### CODE REVIEW

**Comment by benjycui** [comment]
Ping~ @visvadw

File: components/checkbox/index.en-US.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+category: Components
+type: Form Controls
+title: Checkbox
+---
+
+Checkbox.
+
+## When To Use
+
+- Used to select serveral values in multiple options.
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: - Used for selecting several values from multiple options.

File: components/checkbox/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- If you just use only one checkbox, it is the same as `Switch` to switch between two states. The difference is that `Switch` will trigger state change directly, but `Checkbox` just marks state changed and need to be submitted.
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: - If you use only one checkbox, it is the same as using `Switch` to toggle between two states. The difference is that `Switch` will trigger state change directly, but `Checkbox` just marks the state as changed and this needs to be submitted.

File: components/checkbox/index.en-US.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## API
+
+### Checkbox
+
+| Property      | Description             | Type      | Default  |
+|----------|------------------|----------|--------|
+| checked | Specifies whether the current is selected | Boolean  | false |
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: | checked | Specifies whether the checkbox is selected. | Boolean  | false |

File: components/checkbox/index.en-US.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+| defaultChecked | Initial whether or not selected | Boolean | false |
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: | defaultChecked | Specifies the initial state: whether or not the checkbox is selected. | Boolean | false |

File: components/checkbox/index.en-US.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+| onChange | The callback function when the state change | Function(e:Event) | - |
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: | onChange | The callback function that is triggered when the state changes. | Function(e:Event) | - |

File: components/checkbox/index.en-US.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+### Checkbox Group
+
+| Property      | Description             | Type      | Default  |
+|----------|------------------|----------|--------|
+| defaultValue | Default selected value | Array | [] |
+| value | Used to set the current selected value| Array | [] |
```
Comment (visvadw) [inline components/checkbox/index.en-US.md]: | value | Used for setting the currently selected value. | Array | [] |

**Comment by visvadw** [comment]
@benjycui Done. Nice job! 👍

**Comment by visvadw** [comment]
@benjycui I would rename `checkbox/demo/controller.md` as `checkbox/demo/controlled.md`.

**Comment by benjycui** [comment]
Yep, `controlled` is better than `controller`, but we must keep the url stable...

**Pull request merged**